test/e2e/gateway/001_path_condition_match_test.go

```diff
@@ -16,135 +16,134 @@
 package gateway
 
 import (
+	. "github.com/onsi/ginkgo"
 	"github.com/projectcontour/contour/test/e2e"
 	"github.com/stretchr/testify/assert"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	gatewayv1alpha1 "sigs.k8s.io/gateway-api/apis/v1alpha1"
 )
 
-func testGatewayPathConditionMatch(fx *e2e.Framework) {
-	t := fx.T()
-	namespace := "gateway-001-path-condition-match"
+func testGatewayPathConditionMatch(namespace string) {
+	Specify("path match routing works", func() {
```

Comment:
changes to tests are mostly caused by the indent, other changes include:
- change test func to be a namespaced test func
- dont create/delete ns in test
- make sure all resources started in ns as much as possible

Reply:
FWIW selecting the "Hide whitespace changes" option in the GitHub diff settings helps a lot with review here